src/main/java/bio/terra/service/snapshotbuilder/SnapshotBuilderService.java

```diff
@@ -206,51 +214,77 @@ private EnumerateSnapshotAccessRequest convertToEnumerateModel(
     return enumerateModel;
   }
 
-  // Hardcoded stubbed data, will remove once query is implemented.
-  private static SnapshotBuilderConcept createConcept(int id, String name, boolean hasChildren) {
-    return new SnapshotBuilderConcept().id(id).name(name).count(100).hasChildren(hasChildren);
-  }
+  record ParentQueryResult(int parentId, int childId, String childName) {
+    static List<ParentQueryResult> aggregateBigQuery(TableResult result) {
```

Comment:
In general I think utils classes aren't a good way to organize code and  in this case I'd like to keep the code that reads the output of the query close to the code that generates the query. Since it's generating a `ParentQueryResult` it makes sense to put it inside that class too IMO.